output-comparator/src/test/java/org/mobilitydata/gtfsvalidator/outputcomparator/io/NoticeSummaryTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2020 MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.outputcomparator.io;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mobilitydata.gtfsvalidator.notice.SeverityLevel;
+
+@RunWith(JUnit4.class)
+public class NoticeSummaryTest {
+
+  private static NoticeSummary createNoticeAggregate(
+      String code,
+      SeverityLevel severityLevel,
+      int totalNotices,
+      List<Map<String, Object>> notices) {
+    return new NoticeSummary(code, severityLevel, totalNotices, notices);
+  }
+
+  private static boolean testEquality(
+      NoticeSummary firstNoticeSummary, NoticeSummary otherNoticeSummary) {
+    return firstNoticeSummary.equals(otherNoticeSummary);
+  }
+
+  @Test
+  public void equals_sameNotices_true() {
+    assertThat(
```

Comment:
You have an overridden NoticeSummary.equals method. Can you call assertThat(a).isEqualTo(b)?